nebula3/gclient/net/SessionPool.py

```diff
@@ -86,6 +86,8 @@ def init(self, configs=None):
                 configs, SessionPoolConfig
             ), 'wrong type of SessionPoolConfig, try this: `from nebula3.Config import SessionPoolConfig`'
             self._configs = configs
+        else:
```

Comment:
this was added during self-reflection, previously my attempt to support omitting the session pool config was not completed